internal/vehicle/ford.go

```diff
@@ -17,12 +17,12 @@ import (
 )
 
 const (
-	fordAuth             = "https://fcis.ice.ibmcloud.com"
-	fordAPI              = "https://usapi.cv.ford.com"
-	fordVehicleList      = "https://api.mps.ford.com/api/users/vehicles"
-	fordOutdatedAfter    = 5 * time.Minute       // if returned status value is older, evcc will init refresh
-	fordMaxRefreshTrials = 20                    // max trials to get status after refresh, poll interval is 1.5s, i.e. timeout = maxTrials * 1.5s
-	fordTimeFormat       = "01-02-2006 15:04:05" // time format used by Ford API, time is in UTC
+	fordAuth           = "https://fcis.ice.ibmcloud.com"
+	fordAPI            = "https://usapi.cv.ford.com"
+	fordVehicleList    = "https://api.mps.ford.com/api/users/vehicles"
+	fordOutdatedAfter  = 5 * time.Minute       // if returned status value is older, evcc will init refresh
```

Comment:
Müsste man hier nicht egtl immer cc.Timeout nehmen? Solange landet es ja eh im Cache?

Reply:
Falls du cc.Cache meinst: selbst wenn EVCC den Refresh eingeleitet hat, ist der Status beim Cache-Update etwas jünger (z.B. 19min35s bei cc.Cache = 20min), da der Refresh etwas dauert. 
Hatte ursprünglich etwas wie cc.Cache/2, aber das erschien mir noch wahlloser als die 5 Minuten, die bei mir in der Praxis ganz gut funktionieren. Dann kann es immerhin nicht passieren, dass man bei cc.Cache = 20min einen 17min alten Status bekommt, weil das Auto von sich aus zwischendurch ein Update gesendet hat.

Reply:
Ja, macht sinn.